models/topics.models.js

```diff
@@ -64,3 +64,24 @@ exports.insertCommentByArticleId = (body, username, article_id) => {
       return rows[0];
     });
 };
+
+exports.removeCommentById = (comment_id) => {
+  return db
+    .query(`DELETE FROM comments WHERE comment_id = $1 RETURNING*;`, [
```

Comment:
I think you should have a space between RETURNING and *
Nice use of $ syntax to prevent SQL injection 👍